src/components/CoinsSelectAutocomplete/CoinsSelectAutocomplete.tsx

```diff
@@ -423,9 +34,9 @@ export function CoinsSelectAutocomplete({
   return (
     <SelectAutocomplete
       Label="Currency"
-      HelperText={<BackToHomeHelperText env={safeBitgoEnv} />}
+      HelperText={helperText}
```

Comment:
In a future PR I will have two coin selects in the same form. It's really weird having to display this `BackToHomeHelperText` twice in the same form.